build.gradle

```diff
@@ -7,14 +7,15 @@ buildscript {
     dependencies {
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
         classpath 'com.github.maiflai:gradle-scalatest:0.19'
+        classpath "com.github.ben-manes:gradle-versions-plugin:0.27.0"
     }
 }
 
 ext {
-    circeVersion = "0.9.3"
-    metricsVersion = "2.4.4"
-    http4sVersion = "0.18.15"
-    monixVersion = "3.0.0-RC1"
+    circeVersion = "0.13.0"
+    metricsVersion = "2.6.6"
+    http4sVersion = "0.21.1"
+    monixVersion = "3.1.0"
```

Comment:
Please add note this is used only in tests.

Reply:
Done. Added comment.